pkg/microservice/aslan/core/common/service/webhook/controller.go

```diff
@@ -143,6 +143,8 @@ func removeWebhook(t *task, logger *zap.Logger) {
 		t.doneCh <- struct{}{}
 		return
 	}
+
+	logger = logger.With(zap.String("hookID", webhook.HookID))
```

Comment:
logger.infof?

Reply:
it is a structured log. This line means adding a new field "hookID" to all logs below